commands.go

```diff
@@ -183,6 +201,92 @@ func (cmd *MonitorStatus) SetValue(result *Result) error {
 	return nil
 }
 
+// FreezeFrame represents a command that checks the DTC that caused required 
+// freeze frame data storage
+// 
+// Min 0x0000
+// Max 0xFFFF
+type FreezeFrame struct {
+	BaseCommand
+	FrameData string
+}
+
+// NewFreezeFrame creates a new command that contains DTC freeze frame stored data
+func NewFreezeFrame() *FreezeFrame {
+	return &FreezeFrame{
+		BaseCommand{2, 2, "freeze_frame"},
+		"0000",	
+	}
+
+}
+
+// SetValue processes the byte array value into the right unsigned integer
+// value.
+func (cmd *FreezeFrame) SetValue(result *Result) error {
+	expAmount := 2
+	payload := result.value[2:]
+	amount := len(payload)
+
+	if amount != expAmount {
+		return fmt.Errorf(
+			"Expected %d bytes of payload, got %d", expAmount, amount,
+		)
+	}
+	
+	cmd.FrameData = string(payload[:])
+	
+	return nil
+}
+
+// ValueAsLit retrieves the value as a literal representation.
+func (cmd *FreezeFrame) ValueAsLit() string {
+	return fmt.Sprintf(
+		"Freeze frame data: %X", cmd.FrameData,
+	)
+}
+
+// FuelSystemStatus represents a command that reports the fuel system status
+type FuelSystemStatus struct {
+	BaseCommand
+	FuelSystem1 string
+	FuelSystem2 string
```

Comment:
Like I mentioned above on `FreezeFrame` the data needs to be decoded. 

The status for [each fuel system is bit encoded.](https://en.wikipedia.org/wiki/OBD-II_PIDs#Service_01_PID_03). 

Also, there is no guarantee that fuel system 2 exists, so it should be optional.